Docs-5685-partner-manager-EA-mc (#33)
mulesoft/docs-partner-manager: modules/ROOT/pages/index.adoc

endif::[]
:keywords: b2b, edi, portal

Anypoint Partner Manager allows Business-to-Business (B2B) message exchange using APIs and Electronic Data Interchange (EDI) formats.

== Overview

Enterprises use Anypoint Partner Manager to carry out business transactions such as purchasing goods from one another.

> These transactions are typically sent over transports (such as HTTP(s), SFTP, and exchanges such as AS2) and use message formats such as X12 and JSON.

Reviewer: ```suggestion
They are typically transmitted over transports (such as HTTP(s), SFTP, and exchanges such as AS2) and use message formats such as X12 and JSON.
```

All of this is integrated into the Mule runtime.

> Use Anypoint Partner Manager to set up, manage, and monitor message flows for the transactions between you and your business partners.

Reviewer: ```suggestion
Use Anypoint Partner Manager to set up, manage, and monitor the flow of messages between you and your business partners.
```

> Message flows are the path along which a message travels from a source (that is transformed by a map) and is routed to its target.

Reviewer: ```suggestion
Create message flows, the paths along which a message travels from its source, through a transformation map, to its target.
```

> A transmission is a single communication event where there is a transfer of a payload (that can contain multiple messages). The ability to track and report on this drives greater operational efficiency while managing B2B transactions.

Reviewer: ```suggestion
Monitor each transmission, the transfer of a payload and the validation of the messages it contains. The ability to track and report on this drives greater operational efficiency while managing B2B transactions.
```

With Anypoint Partner Manager you can:

* Perform technical on-boarding of your partners.

* Set up message flows to receive messages between your systems and your partners.

> * View transmission information for all message flows.

Reviewer: ```suggestion
* View meta information, validation errors, and all messages included with each transmission payload.
```

// * Set up message flows to receive and send messages between your systems and your partners.
// * View and monitor how partner messages are being processed in the system in case of failure in order to take corrective action.

Partner Manager can be used by different people in your organization.

> For example, those that design and test message flows (message flow developers), those who manage an activity that is part of production (operations managers), and those who optimize back end services and performance (IT managers).

Reviewer: ```suggestion
For example, those that design and test message flows, those who manage activity in your production environment, and those who optimize backend services and performance.
```
